earth_enterprise/tutorial/FusionTutorial/download_tutorial.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+if [ ! -d Imagery ]; then
+	wget http://data.opengee.org/FusionTutorial-Full.tar.gz -O /tmp/FusionTutorial-Full.tar.gz
+	tar -xvzf /tmp/FusionTutorial-Full.tar.gz -C .
```

Comment:
It would be nice if this script could run from any location but still download the files to the directory that the script is in.  For example, you could call it from a parent directory (`tutorial/fusion/download_tutorial.sh`) and it would download the files to the directory that the script is in.  You can get the directory the script is in by running something like SCRIPTDIR=\`dirname $0\`.